mcs_benchmark_data/cli/commands/create_benchmark_pipeline_command.py

```diff
@@ -18,52 +18,47 @@ class CreateBenchmarkPipelineCommand(_Command):
     Creates the directories and files necessary for a benchmark pipeline
     """
 
-    def add_arguments(self, arg_parser: ArgParser, add_parent_args):
-        arg_parser.add_argument(
-            "--benchmark-name", help="name of the new benchmark (in snake_case)"
-        )
-        arg_parser.add_argument(
-            "--using-test-data",
-            help="true if using truncated data for testing (in the test_data directory)\nalters the test file input path",
-        )
+    def __init__(self, args: dict, **kwds):

```

Comment:
This is not correct. You'll get benchmark_name: str as a keyword parameter. So
```
def __init__(self, *, benchmark_name: str, using_test_data: bool, ..., **kwds):
```

Reply:
Is it just not correct in practice? It worked in my test. Is that because it is pulling args globally?